common/generate_bundle_base

```diff
@@ -9,13 +9,19 @@ declare -a overlay_opts=()
 # Catch any changes/additions to master opts
 update_master_opts ${MOD_PASSTHROUGH_OPTS[@]}
 
+type=$(get_cloud_type)
 vip_start=${MASTER_OPTS[VIP_ADDR_START]}
+cidr=""
 if [[ -z $vip_start ]] && [[ -e ~/novarc ]]; then
```

Comment:
Think you can simply change this by changing:
```
if [[ -z $vip_start ]] && [[ -e ~/novarc ]]; then
```

to:
```
if [[ -z $vip_start ]] && [[ -e ~/novarc ]] && [[ $type = openstack ]]; then
```

Then you wouldn't check twice inside the if. And can also drop the `cidr=""` too.

Reply:
hi pon, I made the modifications as you suggested, 

if [[ -z $vip_start ]] && [[ -e ~/novarc ]] && [[ $type = openstack ]]; then

and also added the following code to improve it. thanks.

elif [[ -z $vip_start  ]]; then
    echo "ERROR: No VIP start address provided with --vip-addr-start for the $type cloud."
exit 1